administrator/modules/mod_version/services/provider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  mod_version
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
```

Comment:
```suggestion
\defined('_JEXEC') or die;
```
Same as in the other PR #42827 : New code style rules in 5.0-dev and 5.1-dev. This suggestion will fix PHPCS failing in Drone.

Reply:
Fixed